code/modules/mining/materials.dm

```diff
@@ -127,11 +127,16 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 	var/sheettype=null
 	var/cointype=null
 	var/value=0
+	var/color
```

Comment:
aren't you a br*t?
![image](https://media.giphy.com/media/fYFTES1dWOY5G/giphy.gif)

Reply:
I have not had enough sleep, will fix all but this, as codebase standard is english simplified, not english traditional

Reply:
further proof that the brits ruin everything

Reply:
color is fine

Reply:
>English (simplified)
Ow

Reply:
AMERICANA GODDAMNIT